test-harness/index.js

```diff
@@ -0,0 +1,270 @@
+const fetch = require('node-fetch');
+const requests = require('./requests');
+
+const { exec } = require('child_process');
+const { makeRequest, constructMasterFileName, readFile } = require('./helpers');
+
+async function waitForPrism(done) {
+  try {
+    await fetch('http://localhost:4010');
+    done();
+  } catch (err) {
+    setTimeout(() => {
+      waitForPrism(done);
+    }, 500);
+  }
+}
+
+async function runTest(req) {
+  const { dynamic, ...request } = req;
+
+  const masterFileName = constructMasterFileName(request);
+  const masterFile = readFile(masterFileName);
+
+  const reqRes = await makeRequest(request);
+
+  return {
+    reqRes,
+    masterFile,
+  };
+}
+
+function killPrism(done = () => null) {
+  exec('fuser -k 4010/tcp', () => {
+    done();
```

Comment:
In any case — you should use the `subprocess.kill()` function that the `exec` function is returning when spawning prism

Reply:
Yeah, did that. Didn't work.

Reply:
Ok that's weird, I'll probably look into and see what's wrong.